enhanced-resolve/lib/Storage.d.ts

```diff
@@ -0,0 +1,30 @@
+/// <reference types="node" />
+import { CommonFileSystemMethod } from './common-types';
+
+declare class Storage {
+    duration: number;
+    running: Object;
```

Comment:
Sorry, just learn typescript for two weeks. I've been wondering should i use `{}` object literal type or `Object`. However, i don't think `any` is appropriate as Object is more specific for user to know the type.